### DISCUSSION TITLE
Ability to edit the LLM API keys

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#4726] [Ideas]
### Describe the feature or potential improvement

 It would be really helpful if I could "edit" the LLM API Key settings. We're using the Openai adapter, and whenever we want to add a new custom model, I need to delete and re-add.

**Comment by Chinoman10** [comment]
![image](https://github.com/user-attachments/assets/6f8195a2-463f-4bd3-91a7-cef26a8c1b97)

Would just like to add that I also asked for this on Discord... and even on LinkedIn, in my DMs with Clemens:
![image](https://github.com/user-attachments/assets/6a278c7e-5cf8-44cb-b782-ecc411ca5ac5)

**Reply by marcklingen** [reply]
Thanks for sharing that this is important for you, appreciate it!

**Reply by clemra** [reply]
Thank you, Sergio! We're taking this in -- with christmas around the corner, we may not be able to fix this immediately. Will keep you updated here when it ships!

**Reply by Chinoman10** [reply]
> Thank you, Sergio! We're taking this in -- with christmas around the corner, we may not be able to fix this immediately. Will keep you updated here when it ships!

Totally understand holiday season 🙏  really appreciate you both taking the time to come here and comment!
I've grown to love Langfuse more and more :) keep up the great work!

**Comment by Chinoman10** [comment]
Good news @jannikmaierhoefer , seems like it'll get fixed [soon](https://github.com/orgs/langfuse/discussions/6435#discussioncomment-13056038).
Hoping for this 🤞

**Reply by aantti** [reply]
A potential implementation https://github.com/langfuse/langfuse/pull/7184 ...

**Reply by Chinoman10** [reply]
Thank you for this @aantti !!! 🙏 this is a blessing for us.
Really hope it gets merged soon 🤞

**Comment by maxdeichmann** [comment]
Hi all, we merged this today and it is available on Langfuse Cloud and for self-hosters since [this](https://github.com/langfuse/langfuse/releases/tag/v3.68.0) release. Very much looking forward if you have any feedback on this.

**Reply by maxdeichmann** [reply]
cc @Chinoman10

**Reply by Chinoman10** [reply]
Thank you! 🙏

Not home at the moment, but will 100% dive into this as soon as I get home. Been waiting for this for a while! 😅
